cogs/userinfo.py

```diff
@@ -0,0 +1,35 @@
+from discord.ext import commands
+import discord
+
+class useinfo(commands.Cog, name='Userinfo'):
+    def __init__(self, bot):
+        self.bot = bot
+
+    @commands.command(description = 'Information about a person.')
+    async def userinfo(self, ctx, member: discord.Member = None):
+        if not member:
+            member = ctx.author
+            embed=discord.Embed(title= f"User Information on @{member}", description=f'{member.mention}') 
```

Comment:
There’s a space on both sides of a `=` when it’s a variable definition but no spaces when it’s an keyword argument. So you code should be something like this:
```py
embed = discord.Embed(title=f"User Information on @{member}", description=f'{member.mention}')
```
And why do you use both `"` and `'`?